src/ControlSystem/ControlErrors/Size/Error.hpp

```diff
@@ -35,6 +35,8 @@ struct ErrorDiagnostics {
   double dt_lambda_00;
   double horizon_00;
   double dt_horizon_00;
+  double min_delta_r;
```

Comment:
Is there any documentation of these diagnostics?

Reply:
I plan to write these to disk in the actual control error class itself. There I'll have a full docs list of what everything is that I write